app/js/components/highlight.js

```diff
@@ -1,6 +1,6 @@
 !function(e) {
-    "undefined" != typeof exports ? e(exports) : (window.hljs = e({}),
-    "function" == typeof define && define.amd && define("hljs", [], function() {
+    "undefined" !== typeof exports ? e(exports) : (window.hljs = e({}),
+    "function" === typeof define && define.amd && define("hljs", [], function() {
```

Comment:
Similarly this should go to highlight.js as a patch. Maybe we need to leave `js/components` out of linting.